.travis.yml

```diff
@@ -20,15 +20,15 @@ jobs:
   - stage: Test e2e on private network
     script: make test-e2e
   - stage: Test e2e on public network
-    # disable running this stage for PRs as they do not have access
-    # to the ACCOUNT_PASSWORD env variable needed to unlock the public testnet accounts.
-    # using fork == false may be preferred as it would allow PRs from the origin
-    # to still run, but currently does not work due to a bug
+    # Disable running this stage for PRs as PRs from external contributors
+    # do not have access to ACCOUNT_PASSWORD environment variable anyway.
+    # Also, Travis runs another build called "push" which check
```

Comment:
i thought that it runs push when you actually push to the branch in a repository

Reply:
Yes, it's the other way around 🤦‍♂️